src/client/components/household-information/ChildIncome.jsx

```diff
@@ -12,26 +12,39 @@ import { isBlank, isValidMonetaryValue, validateIfDirty } from '../../utils/vali
  * `onValueChange` - a function with this prototype: (newValue)
  */
 class ChildIncome extends React.Component {
+
+  getRequiredErrorMessage(field) {
+    let result;
+    if (isValidField(isValidMonetaryValue, field)) {
+      if (field.dirty && field.value === '') {
```

Comment:
I feel like we could improve this function if we made us of the validation functions we already have like `validateIfDirty` to keep things consistent, but it's not necessarily a priority.